src/server/api/controllers/channels-controller.js

```diff
@@ -0,0 +1,19 @@
+const knex = require('../../config/db');
+const moment = require('moment-timezone');
+
+const createChannel = async (body) => {
+  await knex('channels').insert({
+    title: body.title,
+    created_at: moment(body.created_at).format(),
+    updated_at: moment(body.updated_at).format(),
+    deleted_at: moment(body.deleted_at).format(),
+  });
+
+  return {
+    successful: true,
```

Comment:
Also return the id of the newly created channel

Reply:
in this case , I have problem to check, could it be like this:
const createChannel = async (body) => {
  const newChannel={
    title: body.title,
  }
  await knex('channels').insert(newChannel);

  return {
    successful: true,
    id:newChannel.id,
    
  };
};
//
if I use 'title' property in place of 'id', it works. But i do not get a result if I use 'id', else would you please give me a hint. I am stuck on this since yesterday.

Reply:
You dont get the id from the body, that is coming from the request. 

You need to get the id from what the insert method returns!